notebooks/05-visualize-distributions-boxplots.ipynb

```diff
@@ -0,0 +1,353 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    for year, value in grouped:
</pre><p>I think <code>year, group</code> woudl be better naming, then below it would be <code>group.values</code> instead of the somewhat awkward <code>value.values</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>